cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -929,7 +929,7 @@
     "accelerator": "DESY-PETRA",
     "authors": [
       {
-        "name": "Allison et al."
+        "name": "Allison, John et al."
```

Comment:
The "et al" part in the name would not be correct. We should have only the full author name there so that the name is searchable uniquely across records regardless of collaborators.

For the "et al" parts, ideally we should expand them, e.g. note 18 shows the full list of authors, so the record here should ideally say:

```yaml
    "authors": [
      {
        "name": "Allison, John"
      },
      {
        "name": "Dittmann, P"
      },
      {
        "name": "Elsen, E"
      },
      {
        "name": "O'Neill, L"
      },
      {
        "name": "Steffen, P"
      },
      {
        "name": "Yamada, S"
      },
    ]
```

Can you please expand all the four instances of "et al" correspondingly?